packages/components/src/InputFile/InputFile.tsx

```diff
@@ -228,12 +258,17 @@ export function InputFile({
   }
 
   async function uploadFile(file: File) {
-    const {
-      url,
-      key = uuidv1(),
-      fields = {},
-      httpMethod = "POST",
-    } = await getUploadParams(file);
+    let params;
+
+    try {
+      params = await getUploadParams(file);
+    } catch {
+      onUploadError && onUploadError(new Error("Failed to get upload params"));
```

Comment:
thinking about if the error messages are OK to live in here or if they should be extracted either to a `messages.ts` or simply to a constant

Reply:
I can extract them to a messages.ts

Reply:
before you do gimme one moment, I'm not seeing any other instances of `messages.ts` in Atlantis - so that actually might not be a pattern we're set up to use

Reply:
ok nevermind that's my bad we don't have that set up for Atlantis

I think a constant in this file will suffice. when we eventually do make such a change it'll be slightly easier to find and replace the user facing string